Engine/ResourceManagement/Importer/ModelImporter.cpp

```diff
@@ -83,13 +83,23 @@ ImportResult ModelImporter::Import(const File& file, bool force) const
 	float time = performance_timer.Read();
 	APP_LOG_SUCCESS("Model %s loaded correctly from assimp in %f ms.", file.file_path.c_str(), time);
 
+	float unit_scale_factor = 1.f;
+	/*for (unsigned int i = 0; i < scene->mMetaData->mNumProperties; ++i)
+	{
+		if (scene->mMetaData->mKeys[i] == aiString("UnitScaleFactor"))
+		{
+			aiMetadataEntry unit_scale_entry = scene->mMetaData->mValues[i];
+			unit_scale_factor = *(double*)unit_scale_entry.mData;
+		};
+	}
+	unit_scale_factor *= 0.01f;*/
```

Comment:
Why is all this code comented?

Reply:
I don't want to apply scale, since artist are having problems with it, but I have not yet decided what to do :D